.env

```diff
@@ -164,9 +162,11 @@ MF_VAULT_UNSEAL_KEY_3=
 MF_VAULT_TOKEN=
 MF_VAULT_CA_NAME=mainflux
 MF_VAULT_CA_ROLE_NAME=mainflux
+MF_VAULT_PKI_PATH=pki
+MF_VAULT_PKI_INT_PATH=pki_int
 MF_VAULT_CA_DOMAIN_NAME=mainflux.com
 MF_VAULT_CA_OU='Mainflux Cloud'
```

Comment:
X.509 cert attributes have their names - for example `CN=Sample Cert, OU=R&D, O=Company Ltd., L=Dublin 4, S=Dublin, C=IE`.

While you use some longer name - like `COUNTRY` instead of `_C` for most of them, you left `_OU` instead writing something like `_ORG_UNIT`.

These kind of inconsistencies hinder the quality of the code.